dashboard2/settings/jptt.md

```diff
@@ -151,3 +151,12 @@ HTML:
 "%"."exampleObject"."exampleNestedHTMLinJS"."nestedHTML" html
 ```
 **Note:** The JSON Path tester tool is **not equipped** to display the nested use case.
+
+#### Skip
+
+Use this processing mode to mark child node not to be translated. Order is important, skip rules must come after the generic * rule.
```

Comment:
Maybe highlight the `.! ` before `skip`. It's not needed for the others so users may go by their gut feeling and omit it.

Reply:
Done.